ci-operator/step-registry/ovn/conf/baremetal-dualstack/ovn-conf-baremetal-dualstack-commands.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -f "${SHARED_DIR}/install-config.yaml" ]; then
+    curl -L https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+    echo "removing networking config if already exists"
+    /tmp/yq e --inplace 'del(.networking)' ${SHARED_DIR}/install-config.yaml
```

Comment:
can we just assume this file is present and skip this if block? if it's not there, then we have a bigger problem
anyway. so with the cli-yq image I think you can just put this single line followed by the new networking config
below

Reply:
Done